packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -91,13 +92,13 @@ abstract class FlutterDriver {
   @visibleForTesting
   factory FlutterDriver.connectedTo({
     FlutterWebConnection webConnection,
-    Browser browser,
+    bool supportTimelineAction,
```

Comment:
This seems like a bit of a roundabout way to specify the web client. Could supportTimelineAction be inferred here?

Reply:
Fixed by wrapping it into FlutterWebConnection.